packages/console/src/pages/EnterpriseSsoDetails/index.tsx

```diff
@@ -69,7 +70,7 @@ function EnterpriseSsoConnectorDetails<T extends SsoProviderName>() {
               .identityProvider
         ));
 
-  const isLoading = !ssoConnector && !requestError;
+  const isLoading = (!ssoConnector && !requestError) || isUserAssetServiceLoading;
```

Comment:
Nit:
```suggestion
  const isLoading = (isLoadingSsoConnectors) || isUserAssetServiceLoading;
```
Since the `useSWR` has updated to v2, the `isLoading` state is returned by the `useSWR` hook.

Reply:
Updated.